govuk-social-providers-only/login/login.ftl

```diff
@@ -75,5 +69,12 @@
                 </ul>
             </div>
         </#if>
+
+        <#if realm.password && realm.registrationAllowed && !usernameEditDisabled??>
+            <div id="kc-registration">
+                <h2 class="heading-medium">${msg("noAccount")}</h2>
+                <span><a href="${url.registrationUrl}">${msg("registerLink")}</a></span>
+            </div>
+        </#if>
```

Comment:
I think it's best to move it down. This is just for the 'social' theme and it looked pretty strange coming up first.

I'm happy to replace the `<span>` with `<p>`. As you say, it currently does nothing. Oddly that `<span>` seems to exist in a few other places as well.